packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/EmailInvoiceDetails.tsx

```diff
@@ -186,7 +186,7 @@ const EmailInvoiceDetailsBase = ({
           </Grid>
         </Toolbar>
       </DialogTitle>
-      <DialogContent className={className + '-Content'}>
+      <DialogContent className={className + '-Content'} sx={{ mt: 4 }}>
```

Comment:
In case, if we have a custom `className`, why should we add hardcoded margins?

Reply:
@radist2s Moving the spacing to the theme will break layout for Material UI theme. We haven't dropped it yet.